BUG - SWAPI I-ALiRT not processing

### Description of the issue

I looked through the logs and we are getting the following error when trying to process SWAPI.

```
[ERROR] IndexError: index 65 is out of bounds for axis 0 with size 63
Traceback (most recent call last):
  File "/var/task/ialirt_ingest.py", line 613, in lambda_handler
    process_algorithms(combined, algorithm_table, algorithm_table_name)
  File "/var/task/ialirt_ingest.py", line 323, in process_algorithms
    result = process_func(combined, calibration_data)
  File "/var/lang/lib/python3.12/site-packages/imap_processing/ialirt/l0/process_swapi.py", line 216, in process_swapi_ialirt
    solution = optimize_pseudo_parameters(
  File "/var/lang/lib/python3.12/site-packages/imap_processing/ialirt/l0/process_swapi.py", line 104, in optimize_pseudo_parameters
    initial_speed_guess = np.sqrt(energy_passbands[max_index]) * Consts.speed_coeff
```

cc @laspsandoval @tech3371

### Steps to reproduce the issue

_No response_

### Expected behavior (What should happen)

_No response_

### Actual behavior (What does happen)

_No response_

### Code Snippet:

```python
Code
```

### Additional notes, affected areas, and suggested fixes

_No response_

--- a/imap_processing/ialirt/l0/process_swapi.py
+++ b/imap_processing/ialirt/l0/process_swapi.py
@@ -18,6 +18,8 @@ from imap_processing.swapi.l2.swapi_l2 import SWAPI_LIVETIME
 
 logger = logging.getLogger(__name__)
 
+NUM_IALIRT_ENERGY_STEPS = 63
+
 
 def count_rate(
     energy_pass: float, speed: float, density: float, temp: float
@@ -191,6 +193,8 @@ def process_swapi_ialirt(
         )
 
     raw_coin_count = process_sweep_data(grouped_dataset, "swapi_coin_cnt")
+    # Subset to only the relevant I-ALiRT energy steps
+    raw_coin_count = raw_coin_count[:, :NUM_IALIRT_ENERGY_STEPS]
     raw_coin_rate = raw_coin_count / SWAPI_LIVETIME
     count_rate_error = np.sqrt(raw_coin_count) / SWAPI_LIVETIME
 
@@ -208,10 +212,12 @@ def process_swapi_ialirt(
         & (calibration_lut_table["Sweep #"] == 2)
     ]
     if subset.empty:
-        energy_passbands = np.full(63, np.nan, dtype=np.float64)
+        energy_passbands = np.full(NUM_IALIRT_ENERGY_STEPS, np.nan, dtype=np.float64)
     else:
         subset = subset.sort_values(["timestamp", "ESA Step #"])
-        energy_passbands = subset["Energy"][0:63].to_numpy().astype(float)
+        energy_passbands = (
+            subset["Energy"][:NUM_IALIRT_ENERGY_STEPS].to_numpy().astype(float)
+        )
 
     solution = optimize_pseudo_parameters(
         raw_coin_rate, count_rate_error, energy_passbands

